### PR TITLE
fix: '\x00' Issue in Python SDK Precompiled SQL Inserts

### PR DESCRIPTION
**Pull request by yht520100**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix

* **What is the current behavior?** (You can also link to an open issue here)
* https://github.com/4paradigm/OpenMLDB/issues/3779
Resolve the issue of extra '\x00' characters in string fields when inserting data using precompiled SQL in the Python SDK.

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: python/openmldb_sdk/openmldb/dbapi/dbapi.py [hunk 1]
```diff
@@ -291,44 +291,50 @@ def execute(self, operation, parameters=()):
...
+                continue
             if isinstance(row, tuple):
-                if isinstance(row[i], str):
-                    str_size += len(row[i])
+                col_value = row[i]
             elif isinstance(row, dict):
                 if name not in row:
                     raise DatabaseError("col {} data not given".format(name))
                 if row[name] is None:
                     if schema.IsColumnNotNull(idx):
```
Comment (vagetablechicken) [inline python/openmldb_sdk/openmldb/dbapi/dbapi.py]: tuple need check not null too?

Comment (yht520100) [reply python/openmldb_sdk/openmldb/dbapi/dbapi.py]: ok，updated

Comment (vagetablechicken) [reply python/openmldb_sdk/openmldb/dbapi/dbapi.py]: > ok，updated

I don't see any change about check not null when row is an instance of tuple, you only check when row is dict

Comment (yht520100) [reply python/openmldb_sdk/openmldb/dbapi/dbapi.py]: updated, please review

Comment (yht520100) [reply python/openmldb_sdk/openmldb/dbapi/dbapi.py]: > > ok，updated
> 
> I don't see any change about check not null when row is an instance of tuple, you only check when row is dict

updated, please review

File: python/openmldb_sdk/tests/dbapi_parameterized_test.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+import sys
+import os
+from openmldb.dbapi import connect
+import pytest
+from datetime import datetime
+
+from .case_conf import OpenMLDB_ZK_CLUSTER, OpenMLDB_ZK_PATH
+
+
+class TestOpenmldbDBAPI:
```
Comment (vagetablechicken) [inline python/openmldb_sdk/tests/dbapi_parameterized_test.py]: Test class name should be the same with file name

Comment (yht520100) [reply python/openmldb_sdk/tests/dbapi_parameterized_test.py]: updated

**Review by vagetablechicken — APPROVED** [review]

**Review by aceforeverd — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**